schemas/process.yml

```diff
@@ -23,6 +9,11 @@
     These fields can help you correlate metrics information with a process id/name
     from a log message.  The `process.pid` often stays in the metric itself and is
     copied to the global field for correlation.
+  reusable:
+    top_level: true
+    expected:
+      - parent
```

Comment:
would this mean process.parent.process?

Reply:
No, we would have process and parent.process.  Moving parent to its own top-level field makes it simpler to re-use the process fields with the existing ECS generators.  The way the generators currently work, if we reused the process fields under process.parent then we would get process.parent.process.*